tests/integration/query/inline_array/with_filter_all_test.go

```diff
@@ -0,0 +1,305 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package inline_array
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestQueryInlineStringArrayWithAllFilter(t *testing.T) {
```

Comment:
suggestion: we try to stick to the following test naming convention:
`<Category>_<Condition|Prerequisite>_<Result|Expectation>`
In this case it could be 
`TestQueryInlineStringArray_WithAllFilter_Succeeds`

Reply:
nitpick: I did think we agreed to not force that suggestion upon developers though, especially the last block.

Reply:
I agree that that last part is optional

Reply:
done